pkg/apis/core/validation/validation.go

```diff
@@ -3192,6 +3194,17 @@ func validateContainerCommon(ctr *core.Container, volumes map[string]core.Volume
 		allErrs = append(allErrs, field.Required(path.Child("image"), ""))
```

Comment:
Doesn't this function also run on the pod templates when creating Jobs, Deployments, etc?

Is it acceptable to also fail the creation of those, or just Pods themselves?

Reply:
If the pod was never going to be viable, it seems better to fail early?  On the other hand, if someone was using this as a template (e.g. doing custom admission control as the pod is created to change the template value into a real value) then it would break them.

Do we have any evidence or anecdata?